vision/google/cloud/vision/annotations.py

```diff
@@ -99,7 +99,7 @@ def from_api_repr(cls, response):
     def from_pb(cls, response):
         """Factory: construct an instance of ``Annotations`` from protobuf.
 
-        :type response: :class:`~google.cloud.grpc.vision.v1.\
+        :type response: :class:`~google.cloudvision.v1.\
```

Comment:
Yep, I did a blanket find/replace incorrectly. Just caught it in tox and fixed it.